api/client/utils.go

```diff
@@ -343,3 +357,13 @@ func readBody(stream io.ReadCloser, hdr http.Header, statusCode int, err error)
 	}
 	return body, statusCode, nil
 }
+
+// serverOSFromServerHeader parses the HTTP header returned from the docker
+// daemon to extract the OS of the server. Header format is Docker/Version (OS)
+func serverOSFromServerHeader(serverHeader string) string {
```

Comment:
Would it make more sense to put this into one of our util packages so that it can be used no the daemon side too one day - if the CLI sends a header like this to it?